syntax = "proto3";
package com.panelsw.ca;
option optimize_for = LITE_RUNTIME;
import "ErrorHandling.proto";

message ServciceConfigDetails{

	enum ServiceStart {
		boot = 0;
		system = 1;
		auto = 2;
		demand = 3;
		disabled = 4;

		unchanged = -1;
	}

	enum DelayStart {
		no = 0;
		yes = 1;
		unchanged1 = -1;	
	}

	bytes name = 1;
	bytes commandLine = 2;
	bytes account = 3;
	bytes password = 4;
	ServiceStart start = 5;
	bytes loadOrderGroup = 6;
	repeated bytes dependencies = 7;
	ErrorHandling errorHandling = 8;
	DelayStart delayStart = 9;
	uint32 serviceType = 10;
}
